rust/core-lib/src/tabs.rs

```diff
@@ -19,6 +19,8 @@
 //! This file is called 'tabs' for historical reasons, and should probably
 //! be renamed.
 
+#![allow(clippy::redundant_clone)]
```

Comment:
ditto: this is actually a reasonable catch from clippy, on L526 we call `manifest.clone()` unnecessarily.

Reply:
Does this mean that `manifest.clone()` -> `manifest`?
Looks feasible, feedback later.